### PR TITLE
New redeem script format for the federation

### PR DESCRIPTION
**Pull request by marcos-iov**
*fed:new-erp-script*
*ci:new-erp-script*
*fit:new-erp-script-kelvin-fix*

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java [hunk 1]
```diff
@@ -116,14 +120,20 @@ public int getRsk2BtcMinimumAcceptableConfirmations() {
 
     public Coin getMinimumPegoutTxValueInSatoshis() { return minimumPegoutTxValueInSatoshis; }
 
-    public long getFederationActivationAge() { return federationActivationAge; }
+    public long getFederationActivationAge(ActivationConfig.ForBlock activations) {
+        return activations.isActive(ConsensusRule.RSKIP353) ?
```
[the remark this answers is not in the document]
Comment (marcos-iov) [reply rskj-core/src/main/java/co/rsk/config/BridgeConstants.java:124]: It's using a different value for `federationActivationAge` after RSKIP353 activation

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

File: rskj-core/src/main/java/co/rsk/peg/ErpFederation.java [hunk 2]
```diff
@@ -46,6 +46,7 @@ public ErpFederation(
         // using the given public keys and csv value
         getRedeemScript();
         validateRedeemScript();
+        getStandardRedeemScript();
```
Comment (Vovchyk) [inline rskj-core/src/main/java/co/rsk/peg/ErpFederation.java:49]:
```suggestion
        getStandardRedeemScript(); // NOSONAR
```

**Review by nathanieliov — APPROVED** [review]
LGTM

**Pull request merged**